tutorials/quick_start/README.md

```diff
@@ -15,6 +15,159 @@ in the future.
 - ultralytics: [https://ultralytics.com](https://ultralytics.com)
 - keras-applications: [https://keras.io/api/applications](https://keras.io/api/applications/)
 
+<table>
```

Comment:
Let's remove the link below mobilenet_v2 for now, and push this modification without the notebook.